### DISCUSSION TITLE
npmjs.com auto-appends ".git" to repository urls

### DISCUSSION THREAD

**Discussion by mmmerl** [npm/feedback#795] [Website] [answered]
This behavior will break urls that don't use have ".git" in them. Not all hosted git uses that extension in the url, so some of us end up with 404ed links.

Running `npm repo` via cli opens the correct url for me, but clicking the link on npmjs 404s because of the appended ".git".

I tried to file a support issue and they said i needed to change my package.json to not have ".git" in the url, but I already don't have ".git" in the url.

I understand if the answer is "wontfix, we think hosted git should serve from urls that end in '.git'", but wanted to at least raise the issue and see if fixing this would be considered.

**Comment by monishcm** [comment, marked as answer]
This was already discussed here https://github.com/npm/feedback/discussions/668

We have it our backlog but low on priority with our current focus.
